Touchups from a top-to-bottom walkthrough and live test (#140)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/con_overview-of-the-servicetelemetry-object.adoc

Use the `backends` parameter to control which storage backends are available for storage of metrics and events, and to control the enablement of Smart Gateways, as defined by the `clouds` parameter. For more information, see xref:clouds_installing-the-core-components-of-stf[].

> Currently, Prometheus can be used as the metrics backend, and ElasticSearch as the events backend. Additional backends may be supported in the future.

Reviewer: ```suggestion
Currently, you can use Prometheus as the metrics backend, and ElasticSearch as the events backend. 
```
Applying active voice. 
Removed "Additional backends may be supported in the future" as it's not definite information about the future.
